plugins/kotlin-as-java/src/main/kotlin/converters/KotlinToJavaConverter.kt

```diff
@@ -332,23 +362,33 @@ internal fun DEnum.asJava(): DEnum = copy(
 //    , entries = entries.map { it.asJava() }
 )
 
-internal fun DObject.asJava(): DObject = copy(
+/**
+ * Parameters [excludedProps] and [excludedFunctions] used for rendering companion objects
+ * where some members (that lifted to outer class) are not rendered
+ */
+internal fun DObject.asJava(
+    excludedProps: List<DProperty> = emptyList(),
+    excludedFunctions: List<DFunction> = emptyList()
```

Comment:
Wondering if having a `Set` here would be better, or if these lists are usually so small that it wouldn't make any difference..

Reply:
I've left the `List`. I don't expect a huge amount of entries in the number of properties or functions that can affect performance. Also, the list that passes here was used in the other places (but probably it was possible to painless switch to `Set` even there).